src/components/testnet-badge/testnet-badge.tsx

```diff
@@ -37,7 +37,7 @@ export const TestnetBadgeComponent = (
                 <Text style={props.styles.text}>
                     {translate('App.labels.youAreOn', {
                         blockchain: props.blockchain ? Capitalize(props.blockchain) : '',
-                        networkName: props.networkName
+                        networkName: props.networkName.split('Testnet').join('') // TODO remove spacing on Near  Testnet
```

Comment:
I suggest you do the following: 

```
export const TestnetBadgeComponent = (
    props: IReduxProps & IThemeProps<ReturnType<typeof stylesProvider>>
) => {
    if (props.testNet) {
        const text = [
            ...new Set(
                translate('App.labels.youAreOn', {
                    blockchain: props.blockchain ? Capitalize(props.blockchain) : '',
                    networkName: props.networkName
                }).split(' ')
            )
        ].join(' ');

        return (
            <View testID="testnet-badge" style={props.styles.container}>
                <Text style={props.styles.text}>{text}</Text>
            </View>
        );
    } else {
        return <View />;
    }
};
```

So that you remove all duplicates and you don't have any empty spaces